pkg/templates/notificationtemplates/notification_generator.go

```diff
@@ -1 +1,61 @@
 package notificationtemplates
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("notification-templates")
+
+// NotificationTemplate contains the template subject and content
+type NotificationTemplate struct {
+	Subject string
+	Content string
+}
+
+// GetNotificationTemplates returns a notification subject and body or an error
+func GetNotificationTemplates(name string, assets Assets) (*NotificationTemplate, error) {
+	templates, err := loadTemplates(assets)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get notification templates")
+	}
+	template := templates[name]
+	return &template, nil
```

Comment:
what about also returning a `bool` to say if the template was found?
ie, something like:
```
template, found := templates[name]
return &template, found, nil
```